test/make.sh

```diff
@@ -39,6 +27,14 @@ find_files() {
     -prune -o -type f "$@"
 }
 
+# Create a temporary file in the auto-cleaned up directory while avoiding
+# overwriting TMPDIR for other processes.
+# shellcheck disable=SC2120 # (Arguments may be passed, e.g. maketemp -d)
+maketemp() {
+  TMPDIR="${DELETE_AT_EXIT}" mktemp "$@"
+}
+
+
 # Compatibility with both GNU and BSD style xargs.
 compat_xargs() {
   local compat=()
```

Comment:
compact is not updates according to template https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/test/make.sh#L46-L62

Reply:
Fixed Thank you!